gammapy/maps/geom.py

```diff
@@ -45,9 +45,10 @@ def pix_tuple_to_idx(pix):
         if np.issubdtype(p.dtype, np.integer):
             idx += [p]
         else:
-            p_idx = np.rint(p).astype(int)
-            p_idx[~np.isfinite(p)] = INVALID_INDEX.int
-            idx += [p_idx]
+            with np.errstate(invalid="ignore"):
+                p_idx = np.rint(p).astype(int)
```

Comment:
Minor: ideally the with context only covers the line in question. Or is the warning emitted from all three lines?

Reply:
Indeed. Corrected.